Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-09-13
Initial Package Version: 2.7
Origin: Rawhide
Description: Addes Missing Header

diff -Naur cracklib,2.7.orig/cracklib/crack.h cracklib,2.7/cracklib/crack.h
--- cracklib,2.7.orig/cracklib/crack.h	1970-01-01 00:00:00.000000000 +0000
+++ cracklib,2.7/cracklib/crack.h	2003-09-13 09:50:47.000000000 +0000
@@ -0,0 +1,15 @@
+
+#ifndef CRACKLIB_H
+#define CRACKLIB_H
+
+/* Pass this function a password (pw) and a path to the
+ * dictionaries (/usr/lib/cracklib_dict should be specified)
+ * and it will either return a NULL string, meaning that the
+ * password is good, or a pointer to a string that explains the
+ * problem with the password.
+ * You must link with -lcrack
+ */
+
+extern char *FascistCheck(char *pw, char *dictpath);
+
+#endif
